setup.py

```diff
@@ -1124,11 +1117,58 @@ def print_box(msg):
 
 
 def main():
-    if BUILD_LIBTORCH_WHL and BUILD_PYTHON_ONLY:
+    global BUILD_LIBTORCH_WHL
+    global BUILD_PYTHON_ONLY
+    global PACKAGE_NAME
+
+    BUILD_LIBTORCH_WHL = os.getenv("BUILD_LIBTORCH_WHL", "0") == "1"
+    BUILD_PYTHON_ONLY = os.getenv("BUILD_PYTHON_ONLY", "0") == "1"
+    BUILD_TWO_WHEELS = os.getenv("SPLIT_BUILD", "0") == "1"
+
+    if BUILD_LIBTORCH_WHL and BUILD_PYTHON_ONLY:  # noqa: F823
         raise RuntimeError(
             "Conflict: 'BUILD_LIBTORCH_WHL' and 'BUILD_PYTHON_ONLY' can't both be 1. Set one to 0 and rerun."
         )
 
+    if BUILD_TWO_WHEELS:
+        setup_cmd = sys.argv[1]
+
+        if (
+            setup_cmd == "build"
+            or setup_cmd == "build_ext"
+            or setup_cmd == "sdist"
+            or setup_cmd == "develop"
+        ):
+            raise RuntimeError(
```

Comment:
I'm a bit confusing here, if this rase a RunTimeError, then the rest of the if statement would be skipped, wouldn't it?

Reply:
I guess this is unsupported use case, looking at the Error text.